pkg/profilecreator/profilecreator.go

```diff
@@ -49,6 +48,10 @@ const (
 	YAMLSuffix = ".yaml"
 )
 
+func init() {
+	log.SetOutput(os.Stderr)
```

Comment:
this is not wrong, but seems redundant: https://github.com/sirupsen/logrus/blob/d59e5619da7b597f81da5b56ba4f315daba0a667/logger.go#L84

We can keep this to make it very explicit what we want, though.

Reply:
Yep, we wanted to have it explicitly to show that we are logging to STDERR.

Reply:
@marcel-apf If you'd like me to remove it please let me know and I can remove it

Reply:
I prefer it as "documentation", to show we want it that way on purpose

Reply:
But indeed, why use it twice?

Reply:
Okay, perfect!

Reply:
got it

Reply:
2 different packages